src/app/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -42,10 +43,12 @@ class MyAccountDispatcher extends QueryDispatcher {
      * @returns {Promise<{status: String}>} Reset password token
      * @memberof MyAccountDispatcher
      */
-    forgotPassword(options = {}) {
+    forgotPassword(options = {}, dispatch) {
         const mutation = MyAccount.getForgotPasswordMutation(options);
-        // TODO: WHEN IMPLEMENTING ALWAYS RETURN THAT EMAIL WAS SENT!!!
-        fetchMutation(mutation);
+        fetchMutation(mutation).then(
```

Comment:
This was done just to have an action dispatched? The BE should always respond with success... Was it really required?

Reply:
Well, depends... This was done to turn off the loading indicator. 
Of course, we can always instantly show that password reset was triggered, but I think it is better to show when exactly the process is finished, so the user will know that he will receive email now.

One thing I will also add an error handling here. Like if the server didn't respond at all.